inc/Engine/Media/PreloadFonts/Context/Context.php

```diff
@@ -47,4 +90,22 @@ public function is_mobile_allowed(): bool {
 			&& $this->options->get( 'do_caching_mobile_files', 0 )
 			&& wp_is_mobile();
 	}
+
+	/**
+	 * Get array of fonts to be excluded.
+	 *
+	 * @return array
+	 */
+	public function get_exclusions(): array {
+		$lists      = $this->data_manager->get_lists();
+		$lists      = isset( $lists->delay_js_exclusions ) ? $lists->delay_js_exclusions : [];
+		$exclusions = array_merge( $this->exclusions, $lists );
```

Comment:
Do we need to add more validation here? something like:
```suggestion
		// Validate that delay_js_exclusions is an array.
    if (!is_array($delay_js_exclusions)) {
        $delay_js_exclusions = [];
    }

    // Merge exclusions only if additional exclusions exist.
    $exclusions = empty($delay_js_exclusions)
        ? $this->exclusions
        : array_merge($this->exclusions, $delay_js_exclusions);
```
to use array_merge only when needed, what do u think?